crates/figma_import/src/toolkit_style.rs

```diff
@@ -549,8 +662,6 @@ pub struct ViewStyle {
     pub blend_mode: BlendMode,
 
     pub display_type: Display,
```

Comment:
This is normally considered a layout type (like None means exclude from layout and all subsequent passes).

Reply:
When we parse the Figma node, there is a `visible` property that we look at and if it's false, we skip the node entirely in `transform_flexbox::visit_node()`. So the resulting view tree has no hidden nodes in it. I don't think this field is used at all. 

I think this field, along with a field others like `flex_wrap` and all the grid layout properties, should eventually be used in the layout struct, but currently they aren't used. For now I was thinking to keep it simple and just only put the fields we use directly in a taffy node into `LayoutStyle`.

Reply:
Got it, that makes sense. If we aren't using something in ViewStyle then it's also OK to just delete it. I probably added "display" because I know it also exists in CSS, but since everything is flex, it's not useful.

Reply:
@timothyfroehlich can you delete `display_type` in your protobuf changes? I would make that small change now, but it would mean having to regenerate all of the serialized files again. That is still a bit tedious as I have to do helloworld, tutorial, and then all of the figma files that we have tests for manually, before running `./gradlew fetchAndUpdateFigmaFiles`. It would be great to have that command or something similar update everything automatically.